src/cypress/integration/pages/TradePage.js

```diff
@@ -0,0 +1,79 @@
+class TradePage {
+  getSymbol(symbol) {
+    return cy.get('div[data-testid="tile-' + symbol.replace('/', '') + '"]')
+  }
+  enterNotional(symbol) {
+    return cy.get('div[data-testid="tile-' + symbol + '"] input')
+  }
+
+  getCurrentPrice(symbol, direction) {
+    return cy.get(
+      'div[data-testid="tile-' +
+        symbol +
+        '"] > div > div > div:nth-child(2)  > div:nth-child(2) > div > button[direction=' +
+        direction +
+        '] > div > div'
+    )
+  }
+
+  getDirectionButton(symbol, direction) {
+    return cy.get(
+      'div[data-testid="tile-' +
+        symbol +
+        '"] > div > div > div:nth-child(2)  > div:nth-child(2) > div > button[direction=' +
+        direction +
+        ']'
+    )
+  }
+
+  getRFQButton(symbol) {
+    return cy.get(
+      'div[data-testid="tile-' +
+        symbol +
+        '"] > div > div > div:nth-child(2)  > div:nth-child(2) > div > div:nth-child(4) button'
+    )
+  }
+
+  getRejectButton(symbol) {
+    return cy.get('div[data-testid="tile-' + symbol + '"] button[data-testid=rfqReject]')
+  }
+  getRequoteButton(symbol) {
+    return cy.get(
+      'div[data-testid="tile-' +
+        symbol +
+        '"] > div > div > div:nth-child(2)  > div:nth-child(2) > div > div:nth-child(4) button'
```

Comment:
This is an example of a bad selector. See `getRejectButton`, we should do something similar. If the attribute doesn't exist on the element, you can add it yourself. It will be useful for you to get familiar with how to do this, I can help where necessary.